server/src/main/java/io/druid/server/metrics/MetricsModule.java

```diff
@@ -95,4 +101,40 @@ public MonitorScheduler getMonitorScheduler(
         monitors
     );
   }
+
+  @Provides
+  @ManageLifecycle
+  public JvmMonitor getJvmMonitor(Properties props)
+  {
+    return new JvmMonitor(getDimensions(props));
+  }
+
+  @Provides
+  @ManageLifecycle
+  public JvmCpuMonitor getJvmCpuMonitor(Properties props)
+  {
+    return new JvmCpuMonitor(getDimensions(props));
+  }
+
+  @Provides
+  @ManageLifecycle
+  public SysMonitor getSysMonitor(Properties props)
+  {
+    return new SysMonitor(getDimensions(props));
+  }
```

Comment:
yeah, it reports the system level metrics and will be same for all the peons, I think it will still be useful to have a way to filter system level metrics for a given task in order to trace/debug issues better.  e.g If one of the tasks is not able to ingest properly, it might be useful to filter the network metrics on that node by just selecting the taskID or dataSource.